processing/data_collection/gazette/spiders/pb_campina_grande.py

```diff
@@ -0,0 +1,39 @@
+from dateutil.parser import parse
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
```

Comment:
Just to follow PEP8's rules for import sections
```suggestion
from dateutil.parser import parse

from gazette.items import Gazette
from gazette.spiders.base import BaseGazetteSpider
```